include/zephyr/arch/x86/multiboot.h

```diff
@@ -105,9 +107,10 @@ struct multiboot_mmap {
 
 /* The flags in the boot info structure tell us which fields are valid. */
 
-#define MULTIBOOT_INFO_FLAGS_MEM	(1 << 0)	/* mem_* valid */
-#define MULTIBOOT_INFO_FLAGS_MMAP	(1 << 6)	/* mmap_* valid */
-#define MULTIBOOT_INFO_FLAGS_FB		(1 << 12)	/* fb_* valid */
+#define MULTIBOOT_INFO_FLAGS_MEM		(1 << 0)	/* mem_* valid */
+#define MULTIBOOT_INFO_FLAGS_CMDLINE		(1 << 2)	/* cmdline* valid */
+#define MULTIBOOT_INFO_FLAGS_MMAP		(1 << 6)	/* mmap_* valid */
+#define MULTIBOOT_INFO_FLAGS_FB			(1 << 12)	/* fb_* valid */
```

Comment:
If you're going to muck with the formatting here, might as well port it to use BIT() instead of the hand-cooked shift.

Reply:
using the `BIT` macro now